components/clarinet-deno/index.ts

```diff
@@ -388,115 +405,115 @@ export namespace types {
     return `0x${hexOctets.join("")}`;
   }
 
-  export function list(val: Array<any>) {
+  export function list(val: Array<unknown>) {
     return `(list ${val.join(" ")})`;
   }
 
   export function principal(val: string) {
     return `'${val}`;
   }
 
-  export function tuple(val: Object) {
+  export function tuple(val: Record<string, unknown>) {
     return `{ ${serializeTuple(val)} }`;
   }
 }
 
 declare global {
   interface String {
-    expectOk(): String;
-    expectErr(): String;
-    expectSome(): String;
+    expectOk(): string;
+    expectErr(): string;
+    expectSome(): string;
     expectNone(): void;
     expectBool(value: boolean): boolean;
     expectUint(value: number | bigint): bigint;
     expectInt(value: number | bigint): bigint;
     expectBuff(value: ArrayBuffer): ArrayBuffer;
-    expectAscii(value: String): String;
-    expectUtf8(value: String): String;
-    expectPrincipal(value: String): String;
-    expectList(): Array<String>;
+    expectAscii(value: string): string;
+    expectUtf8(value: string): string;
+    expectPrincipal(value: string): string;
+    expectList(): Array<string>;
     expectTuple(): Record<string, string>;
   }
 
   interface Array<T> {
     expectSTXTransferEvent(
-      amount: Number | bigint,
-      sender: String,
-      recipient: String,
-    ): Object;
+      amount: number | bigint,
+      sender: string,
+      recipient: string
+    ): ExpectSTXTransferEvent;
     expectFungibleTokenTransferEvent(
-      amount: Number | bigint,
-      sender: String,
-      recipient: String,
-      assetId: String,
-    ): Object;
+      amount: number | bigint,
+      sender: string,
+      recipient: string,
+      assetId: string
+    ): ExpectFungibleTokenTransferEvent;
     expectFungibleTokenMintEvent(
-      amount: Number | bigint,
-      recipient: String,
-      assetId: String,
-    ): Object;
+      amount: number | bigint,
+      recipient: string,
+      assetId: string
+    ): ExpectFungibleTokenMintEvent;
     expectFungibleTokenBurnEvent(
-      amount: Number | bigint,
-      sender: String,
-      assetId: String,
-    ): Object;
+      amount: number | bigint,
```

Comment:
Naive question, is moving from `String` to `string` and `Number` to `number`  going to be backward compatible with current developer codebases?

Reply:
Yes, it'll probably be mostly transparent.

It could lead to changes in the way that it fixes type checking issues such as:
```ts
const n1: Number = 1;
n1 + 1; // Error: Operator '+' cannot be applied to types 'Number' and 'Number'.

const n2: number = 1;
n2 + 1; // Ok
```